app/data/__tests__/team.test.js

```diff
@@ -0,0 +1,134 @@
+    });
+  });
+
+  describe('Comprobando el método hasPlayer()', () => {
+    it('Comprobando que el jugador ya este en el equipo', () => {
+      const jsonObj = readJSONData('./data/sample.json');
+      const team = new Team('CaracasFC');
+      const player = Player.fromDatasetJSONObject(20801, jsonObj);
+      team.addPlayer(player);
+      expect(team.hasPlayer(player)).toBeTruthy();
+    });
+
+    it('Comprobando que el jugador no este en el equipo', () => {
+      const jsonObj = readJSONData('./data/sample.json');
+      const team = new Team('CaracasFC');
+      const player = Player.fromDatasetJSONObject(20801, jsonObj);
+      expect(team.hasPlayer(player)).toBeFalsy();
+    });
+  });
+
+  describe('Comprobando el método getNumberPlayers()', () => {
+    it('Comprobando que devuelve el valor correcto', () => {
+      const jsonObj = readJSONData('./data/sample.json');
+      const team = new Team('CaracasFC');
+      const player = Player.fromDatasetJSONObject(20801, jsonObj);
+      team.addPlayer(player);
+      expect(team.getNumberPlayers()).toBe(1);
+    });
+  });
+
+  describe('Comprobando el método addPlayer()', () => {
+    it('Comprobando que agregue correctamente a un jugador', () => {
+      const jsonObj = readJSONData('./data/sample.json');
+      const team = new Team('CaracasFC');
+      const player = Player.fromDatasetJSONObject(20801, jsonObj);
+      team.addPlayer(player);
+      expect(team.listPlayers.length).toBe(1);    
+    });
+
+    it('Comprobando que el valor de teamValue se modifique al agregar un jugador', () => {
+      const jsonObj = readJSONData('./data/sample.json');
+      const team = new Team('CaracasFC');
+      const player = Player.fromDatasetJSONObject(20801, jsonObj);
+      team.addPlayer(player);
+      expect(team.teamValue).toBe(player.getValue());
+    });
+  });
+
+  describe('Comprobando el método addPlayers()', () => {
+    it('Comprobando que devuelva el valor correcto', () => {
+      const jsonObj = readJSONData('./data/sample.json');
+      const team = new Team('CaracasFC');
+      let ids = [20801, 158023, 190871];
+      let listaJugadores = [];
+      for (let i = 0; i < ids.length; i++) {
+        let player = Player.fromDatasetJSONObject(ids[i], jsonObj);
+        listaJugadores.push(player);
+      }
+      team.addPlayers(listaJugadores);
+      expect(team.listPlayers.length).toBe(ids.length);
+    });
+  });
+  
+  describe('Comprobando el método getTeamOverallQuality()', () => {
+    it('Comprobando que devuelva el valor correcto', () => {
+      const jsonObj = readJSONData('./data/sample.json');
+      const team = new Team('CaracasFC');
+      let ids = [20801, 158023, 190871];
+      let listaJugadores = [];
+      for (let i = 0; i < ids.length; i++) {
+        let player = Player.fromDatasetJSONObject(ids[i], jsonObj);
+        listaJugadores.push(player);
+      }
+      team.addPlayers(listaJugadores);
+      expect(team.getTeamOverallQuality).toBe(279);
```

Comment:
Esto está mal faltan los paréntesis del método
```suggestion
      expect(team.getTeamOverallQuality()).toBe(279);
```

Reply:
resuelto